cmd/clabverter/cli/entrypoint.go

```diff
@@ -9,6 +9,7 @@ import (
 
 const (
 	topologyFile         = "topologyFile"
+	specsFile            = "specsFile"
```

Comment:
I am rooting for shorter arg names, but for this feature I fail to find one.
Maybe though it should be `--topoSpecFile` to be precise what it is about.